ControlsSidebarFullscreen/src/main/java/com/aol/mobile/sdk/testapp/MainActivity.java

```diff
@@ -0,0 +1,132 @@
+import com.aol.mobile.sdk.player.view.PlayerFragment;
+
+public class MainActivity extends AppCompatActivity {
+    public static final String VIDEO_ID = "577cc23d50954952cc56bc47";
+
+    private boolean isFullscreen;
+    private PlayerFragment playerFragment;
+    private View fragmentParent;
+    private RelativeLayout.LayoutParams ogLayoutParams;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        FragmentManager fm = getFragmentManager();
+        playerFragment = (PlayerFragment) fm.findFragmentById(R.id.player_fragment);
+        fragmentParent = (View) playerFragment.getPlayerView().getParent();
+        ogLayoutParams = (RelativeLayout.LayoutParams) playerFragment.getView().getLayoutParams();
+
+        new OneSDKBuilder(getApplicationContext())
+                .create(new OneSDKBuilder.Callback() {
+                    @Override
+                    public void onSuccess(@NonNull OneSDK oneSDK) {
+                        useSDK(oneSDK, playerFragment);
+                    }
+
+                    @Override
+                    public void onFailure(@NonNull Exception e) {
+                        Toast.makeText(getApplicationContext(), "Error Creating SDK", Toast.LENGTH_SHORT).show();
+                    }
+                });
+    }
+
+    private void useSDK(@NonNull OneSDK oneSDK, @NonNull final PlayerFragment playerFragment) {
+        oneSDK.createBuilder()
+                .setAutoplay(true)
+                .buildForVideo(VIDEO_ID, new Player.Callback() {
+                    @Override
+                    public void success(@NonNull Player player) {
+                        playerFragment.getBinder().setPlayer(player);
+
+                        PlayerControlsView playerControlsView = (PlayerControlsView) playerFragment
+                                .getPlayerView().getContentControls();
+
+                        addSideBarButton(playerControlsView.getSidePanel());
+
+                    }
+
+                    @Override
+                    public void error(@NonNull Exception e) {
+                        Toast.makeText(getApplicationContext(), e.getMessage(), Toast.LENGTH_SHORT).show();
+                    }
+                });
+    }
+
+    private void addSideBarButton(@NonNull SidePanel sidePanel) {
+        ImageButton fullscreenButton = makeImageButton(R.drawable.fullscreen);
+        fullscreenButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                final RelativeLayout.LayoutParams curLayoutParams = (RelativeLayout.LayoutParams) playerFragment.getPlayerView().getLayoutParams();
+                if (isFullscreen) {
+                    getSupportActionBar().show();
+
+                    ValueAnimator valueAnimator = ValueAnimator.ofInt(curLayoutParams.height, ogLayoutParams.height);
```

Comment:
Extract this as separate method like `animateToFullscreen` and `animateFromFullscreen`